twentytwenty-blocks/twentytwenty-styles/editor-style-block.css

```diff
@@ -167,48 +167,34 @@ Inter variable font. Usage:
 
 /* CUSTOM COLORS */
 
-:root .has-accent-color {
+.editor-styles-wrapper .has-primary-color,
```

Comment:
Why do we need to target `.editor-styles-wrapper`? I though that enqueueing via `add_editor_style` was supposed to handle this for us?

Reply:
Ah, that's a good question. The non-blocks version of twenty twenty[ enqueues the editor styles using the old hook](https://core.trac.wordpress.org/browser/trunk/src/wp-content/themes/twentytwenty/functions.php#L391), not via `add_editor_style`. Since we don't do that in the block-based theme here, we should probably strip all these out of the editor stylesheet. They're not necessary anymore. 

I'll leave that for a followup PR though.